string entity_name
int16 entity_status